src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -76,6 +93,18 @@
                 :message (str "Invalid level: " level)}))
     :info))
 
+(defn get-service-status-version
+  "Given a params map from a request, get out the service status version and
+   check whether it is valid. If not, throw an error."
+  [params]
+  (when-let [level (params :service-status-version)]
+    (try
+      (Integer. level)
+      (catch NumberFormatException e
```

Comment:
I think we have a parse-int fn in kitchensink that does basically this, but if we don't already have a dep on KS it's probably not worth introducing one just for that.